docs/run/primitives-get-started.mdx

```diff
@@ -45,6 +46,14 @@ observable = SparsePauliOp("Z" * n_qubits)
 print(f">>> Observable: {observable.paulis}")
 ```
 
+The circuit and observable need to be transformed to only use instructions supported by the system. We'll use the transpiler to do this.
```

Comment:
```suggestion
The circuit and observable need to be transformed to only use instructions supported by the system (referred to as *instruction set architecture (ISA)* circuits). We'll use the transpiler to do this.
```

Reply:
@blakejohnson I am not sure why “Instruction Set Architecture” is more clear than "transpiled circuit." To me, ISA highlights the translation stage of transpilation into basis gates, but not the connectivity of the lattice, which is usually a much bigger consideration for getting reliable results.

Reply:
Talking about where transpilation happens is a red herring. We will continue to use transpilation extensively within the Primitives. What we are demanding is that users present circuits to us that comply with the system target. @jaygambetta likes to call these ISA circuits.